plugins/installer/webinstaller/webinstaller.php

```diff
@@ -72,19 +72,20 @@ public function onInstallerViewAfterLastTab()
 		$updatestr2 = JText::_('JLIB_INSTALLER_UPDATE', true);
 
 		$javascript = <<<END
-apps_base_url = '$apps_base_url';
-apps_installat_url = '$apps_installat_url';
-apps_installfrom_url = '$apps_installfrom_url';
-apps_product = '$apps_product';
-apps_release = '$apps_release';
-apps_dev_level = '$apps_dev_level';
-apps_is_hathor = $ishathor;
-apps_installfromon = $installfromon;
-apps_btntxt = '$btntxt';
-apps_pv = '$pv';
-apps_updateavail1 = '$updatestr1';
-apps_updateavail2 = '$updatestr2';
-apps_obsolete = '$obsoletestr';
+var apps_base_url = '$apps_base_url',
+apps_installat_url = '$apps_installat_url',
+apps_installfrom_url = '$apps_installfrom_url',
+apps_product = '$apps_product',
+apps_release = '$apps_release',
+apps_dev_level = '$apps_dev_level',
+apps_is_hathor = $ishathor,
+apps_installfromon = $installfromon,
+apps_btntxt = '$btntxt',
+apps_pv = '$pv',
+apps_updateavail1 = '$updatestr1',
+apps_updateavail2 = '$updatestr2',
+apps_obsolete = '$obsoletestr',
+apps_j25 = $j25;
```

Comment:
It seems this line is generating issue.
Having issue like below with the latest changes.
![web-intaller](https://cloud.githubusercontent.com/assets/2002921/8124052/95056570-10f2-11e5-996c-d4fcaf11cf81.png)